client/lib/two-step-authorization/index.js

```diff
@@ -65,6 +67,70 @@ TwoStepAuthorization.prototype.fetch = function( callback ) {
 	);
 };
 
+TwoStepAuthorization.prototype.postLoginRequest = function( endpoint, data ) {
+	if ( ! this.getTwoStepWebauthnNonce() ) {
+		return Promise.reject( 'Invalid nonce' );
+	}
+
+	const url = 'https://wordpress.com/wp-login.php?action=' + endpoint;
+	// eslint-disable-next-line no-undef
+	const formData = new FormData();
+	const requestData = {
+		client_id: config( 'wpcom_signup_id' ),
+		client_secret: config( 'wpcom_signup_key' ),
+		auth_type: 'webauthn',
+		two_step_nonce: this.getTwoStepWebauthnNonce(),
+		...data,
```

Comment:
putting the ellipsis here allows `data` to overwrite the settings from above. Is that what you intended? 

```
$ node
> var foo = { one: 1 }
undefined
> foo
{ one: 1 }
> var bar = { ...foo, one: 2 }
undefined
> bar
{ one: 2 }
> var baz = { one: 3, ...foo }
undefined
> baz
{ one: 1 }
```

Reply:
It was intended in a previous iteration, but it's not required anymore. Fixed in 53bed2a267f3bf18ea1721d84f8227640c5877eb.